flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/nonha/standalone/StandaloneHaServices.java

```diff
@@ -132,4 +137,23 @@ public LeaderElectionService getJobManagerLeaderElectionService(JobID jobID) {
 			return new StandaloneLeaderElectionService();
 		}
 	}
+
+	@Override
+	public LeaderRetrievalService getRestServerLeaderRetriever() {
```

Comment:
Since `WebMonitorEndpoint` is the instance taking part in leader election currently, this should be called `getWebmonitorLeaderRetriever`.

Reply:
renamed